packages/selenium/src/Resources/branding.js

```diff
@@ -0,0 +1,7 @@
+const brandingConfig = {
+  branding: {
```

Comment:
Should this just be `branding: 'axe-core-selenium-nuget'` as of https://github.com/dequelabs/axe-core/pull/3278 ?

Reply:
You are correct.